Tutorial1-1Basics/src/main/java/com/smarttoolfactory/tutorial1_1basics/chapter2_material_widgets/Tutorial2_10_1BottomSheet.kt

```diff
@@ -151,10 +157,7 @@ private fun FloatingActionButtonComponent(
         },
         backgroundColor = Color(0xffFFA000)
     ) {
-        Icon(
-            Icons.Filled.Navigation, tint = Color.White,
-            contentDescription = null
-        )
+        Icon(Icons.Filled.Navigation, tint = Color.White, contentDescription = "Icon Rotation", modifier = Modifier.rotate(iconRotation))
```

Comment:
Just a suggestion, we could also use `graphicsLayer { }` modifier to rotate icon. This would help in skipping **Composition** phase and only go through **Draw** phase again.